docs/modules/ROOT/pages/migration.adoc

```diff
@@ -0,0 +1,288 @@
+= Migration
+
+To migrate an existing LangChain application to RAGStack and use RAGStack's pinned, tested version of LangChain instead, you need to uninstall LangChain and install RAGStack.
```

Comment:
Could we move this to an aside? I feel like the opening here ought to be something like "Migrating an existing LangChain application to RAGStack is super easy - just change your `requirements.txt` to use `ragstack-ai` and you'll get the benefits of RAGStack, such as <...>". Then we can have an aside that says something like "if you run into problems, you uninstalling your current langchain package could help".

Reply:
Or pyproject.toml depending on how you created your Python project.

Reply:
Uninstalling lanchain is not really needed.
Installing RAGStack will replace the langchain version that was there previously if any.